apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -534,6 +534,7 @@ describe('teacherSectionsRedux', () => {
     it('populates sectionBeingEdited', () => {
       assert.isNull(initialState.sectionBeingEdited);
       const state = reducer(initialState, beginEditingNewSection());
+      state.sectionBeingEdited.createdAt = undefined;
```

Comment:
this doesn't seem right -- the test says it is testing the behavior of `beginEditingNewSection`, but then it modifies the result before checking it for correctness. Either `beginEditingNewSection` should be setting this, or you shouldn't be checking for the inclusion of `createdAt: undefined` below.

Reply:
Is this a situation where we need to freeze time? We want `createdAt` to be set because it's part of the information that's being logged, but we weren't sure how to check that it was the correct time because we don't know the time in the past that it was set.

Reply:
the js never computes the current time -- it only receives `created_at` from the server, correct? if that's the case, then for a test which only covers our js code, this should just be a matter of supplying the correct initial value and then checking it matches that same value here. 

I haven't gone through to figure out exactly where you'd do that, but the definition of `const sections` at the top of this file looks like a good candidate.

Reply:
ohhh, that makes sense! Thanks dave!